mapping/sustainability.yaml

```
@@ -574,7 +574,7 @@
     module: Climate finance
     indicator: Policy coherence
     disclosure format: Select from a list of international commitments and clarify how the project is aligned with it \[codelist, free text to explain\].
-    mapping: See mapping for data point "Policy coherence" in Institutional extension adding `parisAgreement` to the `policyAlighment.policies` array.
+    mapping: See mapping for data point "Policy coherence" in Institutional extension adding `parisAgreement` to the `policyAlignment.policies` array.
```

This change will cause a build error until #428 is merged and 0.9-dev is merged into this branch.

```suggestion
    mapping: See [policy coherence (institutional module)](institutional-policy-coherence).
```